keyboards/atreus/keymaps/dvorak_42_key/keymap.c

```diff
@@ -35,13 +36,18 @@ enum custom_keycodes {
   CLOUD9_GOTO_LINE,
   CLOUD9_NAVIGATE,
 
+  // Windows 10 macros
+  W10_TASKVIEW,
+  W10_WORKSPACE_LEFT,
+  W10_WORKSPACE_RIGHT,
+
 };
 
 // building instructions:
 // make atreus:dvorak_42_key
 
 // flashing instructions:
-// avrdude -p atmega32u4 -c avr109 -U flash:w:atreus_dvorak_42_key.hex  -P COM7
+// avrdude -p atmega32u4 -c avr109 -U flash:w:atreus_astar_dvorak_42_key.hex  -P COM15
```

Comment:
```suggestion
```

`qmk flash` or `make <keyboard>:<keymap>:flash` is the preferred way to flash keymaps.